src/main/java/culturemedia/controller/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
+
+import java.util.*;
+
```

Comment:
renombrar el método find_allVideos a findAllVideos para seguir los estándares de camelCase y generalmente los nombres de los métodos no incluyen guiones bajos.

En el diagrama de clases está el método agregar video, por ende falta en esta clase.

Eliminar la inicialización innecesaria, la variable videos se inicializa como null y luego se asigna el cultureMediaService.findAll(). Se podría declarar y asignar videos directamente.

Validar la excepción VideoNotFoundException, muy parecido al método findAll de CulturotecaServiceImpl.

Usar interfaces en lugar de la implementación.

Cambiar el nombre de algunas variables, clases o paquetes dependiendo su proyecto para que pueda compilar.

Reply:
I already made the changes. Get lucky!